scripts/setup-archlinux.sh

```diff
@@ -43,10 +43,15 @@ PACKAGES+=" lua" # Needed to build luarocks package.
 PACKAGES+=" python-recommonmark" # Needed for LLVM-8 documentation.
 PACKAGES+=" jre8-openjdk-headless"
 
-# Do not require sudo if already running as root.
-if [ "$(id -u)" = "0" ]; then
+# if command ` sudo ` is not found
+if [ ! -x "$(command -v sudo)" ]
+then	# set SUDO to empty 
```

Comment:
1. Redundant check. `sudo` is installed in all normal desktop & server Ubuntu installations as it is the only official way of getting into superuser account (root password is unset & locked). Also package installation is not possible as non-root user.  Installation command must fail if requirements are not met.

2. Formatting change is unwanted.

3. Comments here are unnecessary too.